build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext.versions = [
-            'compileSdk'         : 27,
+            'compileSdk'         : 28,
             'minSdk'             : 15,
-            'targetSdk'          : 27,
-            'androidGradlePlugin': '3.1.3',
+            'targetSdk'          : 28,
+            'androidGradlePlugin': '3.2.0',
             'supportLibrary'     : '27.1.1',
```

Comment:
❗️should be able to delete the `supportLibrary` extra property since the androidx dependencies are using separate semantic version numbers

Reply:
I removed it and everything still works perfectly